crates/solidity/testing/snapshots/cst_output/Expression/function_call_options/generated/0.4.11.yml

```diff
@@ -5,12 +5,20 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected end of input.
+    Error: Expected .
```

Comment:
I think this is a regression in the error messages?

Reply:
This is a known issue with the conditions at the boundary of the pratt and LL parsers. No solution yet.

Reply:
I suggest bringing back the `end of input` string to `render_error_report()` in `parse_error.rs` if `expectations` is empty (even if temporarily)? Otherwise the error string is confusing.

Reply:
Good idea, will do